modules/@apostrophecms/doc-type/index.js

```diff
@@ -1337,56 +1351,62 @@ module.exports = {
           }
         },
 
-        // `.explicitOrder([ id1, id2... ])` causes the query to return values
-        // in that order, assuming the documents with the specified ids exist.
-        // If a doc is not mentioned in the array it will
-        // be discarded from the result. Docs that
-        // exist in the array but not in the database are
-        // also absent from the result.
-        //
-        // As a second argument you may optionally specify a property name
-        // other than `_id` to order on.
-
-        explicitOrder: {
-          set(values, property) {
-            property = property || '_id';
-            query.set('explicitOrder', values);
-            query.set('explicitOrderProperty', property);
+        // `._id([ id1, id2... ])` causes the query to return only those
+        // documents, and to return them in that order, assuming the documents
+        // with the specified ids exist. All documents are fetched in the
+        // same locale regardless of the locale suffix of the _id. If
+        // no locale can be determined via query parameters, the locale is
+        // inferred from the first _id.
+
+        _id: {
+          set(values) {
+            if (Array.isArray(values)) {
+              query.set('_id', values);
+            } else if (values) {
+              query.set([ values ]);
+            }
+          },
+          launder(values) {
+            return self.apos.launder.ids(values);
           },
           finalize() {
-            if (!query.get('explicitOrder')) {
+            if (!query.get('_id')) {
               return;
             }
             const criteria = {};
-            const values = query.get('explicitOrder');
-            const property = query.get('explicitOrderProperty');
+            let values = query.get('_id');
             if (!values.length) {
               // MongoDB gets mad if you have an empty $in
-              criteria[property] = { _id: null };
+              criteria._id = { _id: null };
               query.and(criteria);
               return;
             }
-            criteria[property] = { $in: values };
+            if (self.isLocalized()) {
+              const parts = values[0].split(':');
+              if (parts.length > 1) {
+                values = values.map(value => self.inferIdLocaleAndMode(query.req, `${value.split(':')[0]}:${parts[1]}:${parts[2]}`));
+              }
+            }
+            criteria._id = { $in: values };
             query.and(criteria);
-            query.set('explicitOrderSkip', query.get('skip'));
-            query.set('explicitOrderLimit', query.get('limit'));
+            query.set('_idSkip', query.get('skip'));
```

Comment:
If we're specifying IDs, why are we inviting skipping?

Reply:
Just because you know you are interested in 200 ids in a specific order doesn't mean you don't still want pagination to avoid exhausting memory by carrying out all the relationship nested queries for 200 docs when you're only showing "page two." More generally, if a query builder can avoid breaking other query builders, it should do that.

Reply:
makes sense.